cmd/pebble/cmd_boot_firmware.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2022 Canonical Ltd
```

Comment:
If you have two variants of cmd_boot_firmware*, then the bootstrap package does not have to be termus specific, which feels nicer, as its very generic PID1 behaviour?